docs/configuration.md

```diff
@@ -1,15 +1,24 @@
 # Configure the bot
 
-This page explains how to configure your `config.json` file.
+This page explains how to configure your configuration file.
```

Comment:
"configure your configuration", hmm

Reply:
THAT's why we had `config.json` there i think ... :D 

maybe "This page explains how to tune your configuration file." ? (but it's not very well either)

Reply:
I reworked this part of the docs, tried to make it more friendly for novice users...

Pls review.

Reply:
Nicely worked around that odd phrase! :+1: